### ISSUE TITLE
SFTPgo image Selector not changing container image to v2.5.4-plugins

### ISSUE THREAD

**Issue by LightningManGTS** [truenas/charts#2030]
In the last and most current version (1.2.1 and 1.2.2) changing from default to plugins does not change the container image in use from drakkan/sftpgo:v2.5.6 to drakkan/sftpgo:v2.5.4-plugins

![image](https://github.com/truenas/charts/assets/3919521/8b020435-b9aa-4567-97d2-0afb4297a945)

![image](https://github.com/truenas/charts/assets/3919521/3d799dbd-c133-435a-a187-7bf5fc24fc12)

![image](https://github.com/truenas/charts/assets/3919521/03f89ef1-ccde-41b9-871f-74fc050f1deb)

**Comment by LightningManGTS** [comment]
While were in here, it may not be a bad idea to update the plugin container image from 2.5.4-plugins to 2.5.6-plugins 😉

**Comment by stavros-k** [comment]
Fixed the issue that was skipped.
I'll let the bot to update it after its merged, so we can verify it works!
Thanks

**Comment by LightningManGTS** [comment]
> Fixed the issue that was skipped. I'll let the bot to update it after its merged, so we can verify it works! Thanks

Fair enough, either way thanks for the quick fix! Looking forward to the merge!
